db/migrate/20220712023616_create_games.rb

```diff
@@ -1,7 +1,6 @@
 class CreateGames < ActiveRecord::Migration[7.0]
   def change
     create_table :games do |t|
-      t.string :id
```

Comment:
This is fine for now, but remember that I previously mentioned that we should avoid editing existing migrations?

A migration is a db change. If we had a production environment, this migration would have already been run on that db and editing it would not do anything (or it could cause issues if we tried to roll it back). The more robust way to do this would be to create another migration `RemoveIdFromGames`

But we've already dropped the games table so it shouldn't matter. Just something to be aware of